lib/src/usage.dart

```diff
@@ -133,118 +130,103 @@ class Usage {
     return result;
   }
 
-  String getAllowedTitle(Option option, String allowed) {
+  String _allowedTitle(Option option, String allowed) {
     var isDefault = option.defaultsTo is List
         ? option.defaultsTo.contains(allowed)
         : option.defaultsTo == allowed;
     return '      [$allowed]' + (isDefault ? ' (default)' : '');
   }
 
-  void calculateColumnWidths() {
+  List<int> _calculateLineWidths() {
```

Comment:
I think `_calculateColumnWidths` is a better name.

Reply:
Oops, I mistyped when I did the rename to make this private. Done.